src/store/loginName/loginSlice.ts

```diff
@@ -6,27 +6,55 @@ export const getLogin = createAsyncThunk(
   async (value: {email: string, password: string}) => {
     const response = await fetch(`${BACKEND_URL}/login?email=
     ${value.email}&password=${value.password}`, {method:"POST"});
```

Comment:
hm, we shouldn't pass password into the url - it's not secure.

Right way - to put in into the request body.

issue now for you'r PR, but, please, send it to the your team lead - we need backend changes